src/test/java/com/datastax/cdm/schema/CqlTableTest.java

```diff
@@ -0,0 +1,27 @@
+package com.datastax.cdm.schema;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.datastax.cdm.cql.CommonMocks;
+import com.datastax.oss.driver.api.core.ConsistencyLevel;
+
+class CqlTableTest extends CommonMocks {
+
+    @Test
+	void testCL() {
+        assertEquals(CqlTable.mapToConsistencyLevel("LOCAL_QUORUM"), ConsistencyLevel.LOCAL_QUORUM);
```

Comment:
why one these 2 are in upper case and others in lower case?

Reply:
If you look at all the options, I have used many mixed formats (upper, lower & both).